src/Model/Verify.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * @file src/Model/Verify.php
+ */
+namespace Friendica\Model;
+
+use Friendica\Database\DBM;
+use Friendica\Util\DateTimeFormat;
+use dba;
+
+/**
+ * Methods to deal with entries of the 'verify' table.
+ */
+class Verify
```

Comment:
Same naming question as for the table: if it's only use is for OpenWebAuth then it should mention it. Suggestion: `OpenWebAuthToken`.